Update GCP credentials docs (#6837)
wazuh/wazuh-documentation: source/cloud-security/gcp/prerequisites/credentials.rst

To do this, it is recommended to create a service account with the Pub/Sub or Storage permissions and then create a key. It is important to save this key as a JSON file as it will be used as the authentication method for the GCP module.

Creating a custom role
----------------------

The ``gcp-bucket`` module requires permissions to access storage buckets and objects. These permissions were originally assigned using the **Storage Legacy Bucket Writer** role, but it is no longer available for selection, so it will be necessary to create a new one.

> To do so, go to the **Roles** section and click on **Create Role**.
> 
> Once you have chosen a name, click on **Add Permissions** and on the search bar type **Storage Legacy Bucket Writer**. After that, select the permissions:
> 
>    - ``storage.bucket.get``
>    - ``storage.objects.create``
>    - ``storage.objects.delete``
>    - ``storage.objects.list``
> 
> And click on **Create**.
> 
> .. thumbnail:: /images/cloud-security/gcp/gcp-bucket-role.png
>     :align: center
>     :width: 100%

Reviewer: ```suggestion
To do so, follow these steps:

1. Go to the **Roles** section and click on **Create Role**.
2. Once you have chosen a name, click on **Add Permissions**.
3. On the search bar type **Storage Legacy Bucket Writer** and assign the following permissions:

   - ``storage.bucket.get``
   - ``storage.objects.create``
   - ``storage.objects.delete``
   - ``storage.objects.list``

.. thumbnail:: /images/cloud-security/gcp/gcp-bucket-role.png
    :align: center
    :width: 100%

4. Click on **Create**.
```

Also, the different fields could be highlighted in the image.

```

To do this, it is recommended to create a service account with the Pub/Sub or Storage permissions and then create a key. It is important to save this key as a JSON file as it will be used as the authentication method for the GCP module.

Creating a custom role
----------------------

> The ``gcp-bucket`` module requires permissions to access storage buckets and objects. These permissions were originally assigned using the **Storage Legacy Bucket Writer** role, but it is no longer available for selection, so it will be necessary to create a new one.

Reviewer 2: Originally... but it is no longer available..? When? What does all this mean?

> The ``gcp-bucket`` module requires permissions to access storage buckets and objects. These permissions were originally assigned using the **Storage Legacy Bucket Writer** role, but it is no longer available for selection, so it will be necessary to create a new one.

Reviewer 2: ```suggestion
The ``gcp-bucket`` module requires permissions to access storage buckets and objects. To create a role with the required permissions, follow these steps:
```

> To do so, follow these steps:

Reviewer 2: ```suggestion
To create the new role, follow these steps:
```

> To create the new role, follow these steps:

Reviewer 2: ```suggestion
```

> 

Reviewer 2: ```suggestion
```

> 1. Go to the **Roles** section and click on **Create Role**.

Reviewer 2: ```suggestion
#. Go to the **Roles** section and click on **Create Role**.
```

> 2. Choose a name and click on **Add Permissions**.

Reviewer 2: ```suggestion
#. Establish a **Title** and click on **Add Permissions**.
```

> 3. On the search bar, type **Storage Legacy Bucket Writer** and select the following permissions:

Reviewer 2: ```suggestion
#. On the search bar, filter available permissions by typing **Storage Legacy Bucket Writer**. Select the following ones:
```

   - ``storage.bucket.get``
   - ``storage.objects.create``
   - ``storage.objects.delete``
   - ``storage.objects.list``

> 4. Click on **Create**.

Author: In the previous suggestion the 4th point was below the image but it didn't look so good once compiled, so I included this point below the other ones

![screenshot](https://github.com/wazuh/wazuh-documentation/assets/51374959/1bc86cc8-ec27-4d27-b7e9-b5e313d9d4cc)
